index.bs

```diff
@@ -932,9 +928,8 @@ and "timestamp" as arguments.
     The {{Sensor/stop()}} method must run these steps:
 
     1.  If |sensor_instance|.{{[[state]]}} is "idle", then return.
-    1.  Set |sensor_instance|.{{[[state]]}} to "idle".
     1.  Run these sub-steps [=in parallel=]:
-        1.  Invoke [=unregister a sensor object=] with |sensor_instance| as argument.
+        1.  Invoke [=deactivate a sensor object=] with |sensor_instance| as argument.
```

Comment:
Same as [=activate a sensor object=], might be better to run it inside task scope.